### PR TITLE
Convert eduroam to multiple pages

### PR DESCRIPTION
**Pull request by blizzardfinnegan**
As discussed in #8 , having one page for all possible methods is... a bit short sighted. This pull request standardizes documentation to have one page for each method of configuration, from start to finish, including some basic troubleshooting steps. It also includes general configuration instructions, for other users intending on using networking solutions not documented currently.

### CODE REVIEW

File: eduroam.md [hunk 1]
```diff
@@ -8,271 +8,38 @@ layout: default
...
+    - Location of the personal certificate and/or key
+    - Certificate Password: created previously
+7. Optionally, test your network connection.
+
+## Network Management Options
+
+- [Network Manager](./eduroamConfigs/networkManager.md)
+- [nmcli/Network Manager CLI](./eduroamConfigs/nmcli.md)
+- [iwd](./eduroamConfigs/iwd.md)
+- [wpa_supplicant](./eduroamConfigs/wpa_supplicant.md)
```
Comment (MoralCode) [inline eduroam.md:41]: Should these be .md links? Or have no extension? Not sure what makss them clickable on tbe final compiled site

**Review by MoralCode — COMMENTED** [review]
Havrnt had a cnance to fully compile, but heres some initial thoughts from skimming it

File: eduroam.md (hunk 1 above)
Comment (blizzardfinnegan) [reply eduroam.md:41]: Links are clickable in the full site without a file extension (see index.md). I was using autofill to get the pathing right for them, that should be an easy change

File: eduroamConfigs/iwd.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+6. You will then be prompted to create a password to protect your private key. This will be needed to set up eduroam on a new device. If you are unsure, you can use [1Password's public generator](https://1password.com/password-generator/?) which has a "memorable password" mode. The password you choose must be at least six (but no more than 16) characters long and contain a letter, number, and symbol. 
+
+![](/assets/img/eduroam/password.png)
+
+7. Choose a format to download your certificate in. Select PEM format and save the file somewhere you can get to it easily.
+
+![](/assets/img/eduroam/cert-download.png)
+
+8. Click the link on this page to save the RIT CA Cert. You'll also want to have this handy for the next step
```
[the remark this answers is not in the document]
Comment (blizzardfinnegan) [reply eduroamConfigs/iwd.md:53]: Excellent point; I hadn't considered that. I can break that out into another page after lunch today.

**Comment by blizzardfinnegan** [comment]
Certificate generation has been moved to its own file, with links to and from it connected properly (at least, according to Github). Also added certificate generation to the main page, for clarity.

**Review by MoralCode — COMMENTED** [review]
Do you think it would it be better to just have a directory structure like either:

- eduroam.md
- eduroam
  - certificates.md 
  - configs here 

or 

- eduroam
  - index.md
  - certificates.md
  - configs here

(also this includes renaming eduroamConfigs dir to just eduroam) 

i imagine if GH pages allows it, id prefer the second way (assuming it correctly routes a request  for /eduroam (no trailing slash).


also, i haven't really been paying much attention to the content since i've been assuming that its all/mostly just rearranging stuff so let me know if there's any content that you want me to look over in more detail

**Comment by blizzardfinnegan** [comment]
I moved the eduroam main page into the suggested page. At least within GitHub (without looking at Jekyll or using GHPages), it seems to just open the folder itself? I think that's functionality that will have to be tested by someone else.

The content should be reasonably set at this point. I haven't seen any major rendering issues or anything.

**Comment by MoralCode** [comment]
well here goes nothin

**Pull request merged**